domains/transfer/Transfer.API/API/Transfer/Api/Repository/TransferAgreementProposalRepository.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Threading.Tasks;
+using API.Shared.Data;
+using API.Transfer.Api.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Transfer.Api.Repository;
+
+public interface ITransferAgreementProposalRepository
```

Comment:
We normally have our interfaces in a seperate file, i think this should be aligned.
I have no preference whether its in the class or a separate file, as long as we are aligned

Reply:
I think that there should be room to determine this yourself for cases where the interface is small fx